index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script src="js/index.js"></script>
+
+    <h1 id="title">freeCodeCamp Survey Form</h1>
```

Comment:
Hello @admirerbrown to improve the readability of your code i suggest you wrap the h1 in a header tag. html semantics improve the overall readability and maintainability of your code.please make this change.

Reply:
- [ ] Greate job so far 👏. However, I think you should at least have two comments. Please consider adding another comment. You can intentionally introduce some bugs or do something that should be changed fro instance a commented code in the codebase; and request for a change. Actually, the purpose of this exercise is to help you write good and polite comments when reviewing someone's work.
### Note:
Do not forget to explain `why` you requested the change.